docs/howto/grafana-github-auth.md

```diff
@@ -3,6 +3,12 @@
 
 We can enable GitHub authentication against a Grafana instance in order to allow access to the dashboards for hub administrators as well as 2i2c engineers.
 
+```{note}
+We only offer this method of authentication to communities if they want to give
```

Comment:
While reading this I was thinking it might be worth nothing that we use this by default to grant access to all 2i2c staff into Grafanas. Maybe somthing like:

```suggestion
This is the default authentication method for 2i2c staff wanting to visualize the dashboards, but we can also offer this method of authentication to communities only if they want to give
```

Reply:
Yeah. I think to me it is not clear that enabling GitHub auth on a grafana instance and granting 2i2c org access to it actually is the default/something we do every time we deploy _a new cluster_.

I think what we _actually_ do is add the new cluster to the central grafana, and we have GitHub auth for the 2i2c org there.

There's a usage question here of pulling up the specific cluster grafana URL vs finding it in the central grafana. Which path is most used? Do we expect to have GitHub auth for 2i2c enabled on all cluster grafanas, because I don't think it is?

So I think I would reword this further to: "This is the default authentication method for 2i2c staff wanting to visualise the dashboards _on the 2i2c central grafana_. However, we can also offer this method of communication..."

What do you think?

Reply:
@sgibson91, good point! +1 on going with your suggested rewording 👍🏼

Reply:
Thank you!